src/event-stream/event-server.ts

```diff
@@ -484,11 +484,9 @@ async function handleNewAttachmentMessage(msg: CoreNodeAttachmentMessage[], db:
           // case for subdomain
           const subdomains: DbBnsSubdomain[] = [];
           for (let i = 0; i < zoneFileTxt.length; i++) {
-            if (!zoneFileContents.uri) {
-              throw new Error(`zone file contents missing URI: ${zonefile}`);
-            }
             const zoneFile = zoneFileTxt[i];
             const parsedTxt = parseZoneFileTxt(zoneFile.txt);
+            if (Object.entries(parsedTxt).find(e => e[1] === '') != undefined) continue; //if tx has any empty field , skip it
```

Comment:
Can this be a more explicit check, even if it ends up being more verbose? Hard to tell exactly what can trigger this condition.